code/datums/jobs.dm

```diff
@@ -1487,6 +1487,31 @@ ABSTRACT_TYPE(/datum/job/civilian)
 		src.access = get_access("Mail Courier")
 		return
 
+/datum/job/special/stowaway
+	name = "Stowaway"
+	limit = 2
+	wages = 0
+	low_priority_job = TRUE
+	slot_back = list(/obj/item/storage/backpack/satchel/anello)
+	slot_belt = list(/obj/item/crowbar)
+	slot_foot = list(/obj/item/clothing/shoes/brown)
+	slot_jump = list(/obj/item/clothing/under/color/grey)
+	slot_head = list(/obj/item/clothing/head/green)
+	slot_ears = list(/obj/item/device/radio/headset/civilian)
+	slot_poc1 = list(/obj/item/currency/spacecash/fivehundred)
+	slot_poc2 = list(/obj/item/scissors)
+	slot_lhan = list(/obj/item/screwdriver)
```

Comment:
These vars accept lists to randomly pick from, this seems like a great opportunity to randomize the clothes they're given so it's not possible to learn The stowaway outfit and spot them from that.